modules/RTC/BridgeChannel.js

```diff
@@ -359,11 +359,25 @@ export default class BridgeChannel {
                 break;
             }
             case 'LastNEndpointsChangeEvent': {
-                // The new/latest list of last-n endpoint IDs (i.e. endpoints for which the bridge is sending video).
-                const lastNEndpoints = obj.lastNEndpoints;
+                if (!FeatureFlags.isSourceNameSignalingEnabled()) {
```

Comment:
Where does this come from? Bad merge perhaps?

Reply:
Yeah - that'll be a bad merge - I was plagued with them yesterday with these changes that are affecting multiple files - I think going forward we should do them in multiple steps